frontend/src/features/projects/spl/tabs/CloseOutFormTab.tsx

```diff
@@ -47,7 +46,7 @@ const CloseOutFormTab: React.FunctionComponent<ICloseOutFormTabProps> = ({
         <Col md={1}></Col>
         <Col>
           <ProjectNotes
-            field={`notes[${NoteTypes.Comments}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption